packages/react-composites/src/composites/CallWithChatComposite/SidePane.tsx

```diff
@@ -84,6 +85,9 @@ export const EmbeddedPeoplePane = (props: {
   callAdapter: CallAdapter;
   chatAdapter: ChatAdapter;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onChatButtonClick: () => void;
+  onPeopleButtonClick: () => void;
+  isMobile?: boolean;
```

Comment:
This is called `mobileView` elsewhere. Can we continue to call it that for consistency?